test/dataset.json

```diff
@@ -1,7 +1,9 @@
 {
-  // Elasticsearch index will be given this name.
-  "name": "test",
-  // Name of BigQuery field that is primary key. Elasticsearch documents will
-  // have this ID.
+  // Dataset name. This name will be displayed in the Data Explorer UI. Also,
+  // the Elasticsearch index will be given this name.
+  "name": "Insurance data",
+  // Name of BigQuery column that represents primary key. See
+  // https://github.com/DataBiosphere/data-explorer-indexers/tree/master/bigquery#overview
```

Comment:
I'm not sure how this link explains the primary key, and this will break if we ever decide to change that README without remembering to update this. An in-line comment description would probably be fine here.

Reply:
It's from a [PR that's not merged yet](https://github.com/DataBiosphere/data-explorer-indexers/tree/config-info/bigquery#overview). It's a lengthy explanation.

This comment (lines 5 and 6) actually came from that PR. I figured I would update in this PR.